tools/metaquantome/.shed.yml

```diff
@@ -0,0 +1,8 @@
+name: metaquantome
```

Comment:
This should be a suite shed.yml right? Similar to thos of openms?

Reply:
oh, i didn't know about that! how does it look now? https://github.com/galaxyproteomics/tools-galaxyp/blob/ed0e04cd7b710f222424ef4652afb00eceee41e7/tools/metaquantome/.shed.yml

Reply:
You are missing the auto_tool_repositories thing.

Reply:
does it look ok now? I added that in the [latest commit](https://github.com/galaxyproteomics/tools-galaxyp/commit/2fa9b3ce236b45ac8fa2ec258d0f0e1310f96a29)